src/App.js

```diff
@@ -73,6 +76,9 @@ export default function App() {
 
     navigate('/');
   }
+  function handleDeletePlant(Id) {
+    setSavedPlants(savedPlants.filter(card => card._id !== Id));
+  }
```

Comment:
I would recommend to write the parameter without a capital letter. Usually we only start components with a capital letter.
```suggestion
  function handleDeletePlant(id) {
    setSavedPlants(savedPlants.filter(card => card._id !== id));
  }
```